stackanetes/k8s_instance.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, service_name, service_dir):
         self.service_type = service_name.split("-")[0]
         self.service_dir = service_dir
         self._load_service_variables()
-        if not check_if_namespace_exist(CONF.stackanetes.namespace):
+        if not check_if_namespace_exist(self.namespace):
```

Comment:
`check_if_namespace_existS` (although I would prefer to name the method `namespace_exists`)